rpc/context.go

```diff
@@ -45,14 +45,15 @@ const (
 // service.
 func NewServer(ctx *Context) *grpc.Server {
 	var s *grpc.Server
+	maxMsgSize := grpc.MaxMsgSize(128 << 20) // 128 MB
```

Comment:
Maybe this wants to build up a slice of options like we do in the client. I'd make it maxint for now.